internal/cli/up.go

```diff
@@ -136,6 +140,11 @@ func (c *upCmd) run(ctx context.Context, args []string) error {
 
 	logOutput(fmt.Sprintf("Starting Tilt (%s)…", buildStamp()))
 
+	//if --watch was set, warn user about deprecation
+	if c.watchFlagExplicitlySet {
+		logOutput("Flag --watch has been deprecated, it will be removed in future releases.")
```

Comment:
fyi, i think you can also use logger.Get(ctx).Warnf(...) if you want it to show up as a warning in the UI

Reply:
Warnf() seems to be doing something interesting - it prints the warning fine on the TUI, but the keyword "warning" doesn't show up on the Web UI for some reason.

![Screenshot from 2020-06-12 17-56-14](https://user-images.githubusercontent.com/35721712/84510812-ac2f9600-acd6-11ea-85ff-dbc7eb7a3aaf.png)

![Screenshot from 2020-06-12 17-57-04](https://user-images.githubusercontent.com/35721712/84510817-ad60c300-acd6-11ea-85d0-899f2476ef14.png)

Reply:
yep! it has the yellow UI bar in the web, and skips the prefix